embarrassingly-parallel-with-delayed-or-futures.ipynb

```diff
@@ -0,0 +1,512 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<img src=\"http://dask.readthedocs.io/en/latest/_images/dask_horizontal.svg\" \n",
+    "     width=\"30%\" \n",
+    "     align=right\n",
+    "     alt=\"Dask logo\">\n",
+    "\n",
+    "Embarrassingly parallel Workloads\n",
+    "--------------------------------------------------\n",
+    "\n",
+    "This notebook shows how to use Dask to parallelize embarrassingly parallel workloads where you want to apply one function to many pieces of data independently.  It will show three different ways of doing this with Dask:\n",
+    "\n",
+    "1. [dask.delayed](http://dask.pydata.org/en/latest/delayed.html) \n",
+    "2. [concurrent.Futures](https://dask.pydata.org/en/latest/futures.html) \n",
+    "3. [dask.bag](https://dask.pydata.org/en/latest/bag.html)\n",
+    "\n",
+    "This example focuses on using Dask for building large embarrassingly parallel computation as often seen in scientific communities and on High Performance Computing facilities, for example with Monte Carlo methods. This kind of simulation assume the following:\n",
+    "\n",
+    " - We have a function that runs a heavy computation given some parameters.\n",
+    " - We need to compute this function on many different input parameters, each function call being independent.\n",
+    " - We want to gather all the results in one place for further analysis."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Start Dask Client for Dashboard\n",
+    "\n",
+    "Starting the Dask Client will provide a dashboard which \n",
+    "is useful to gain insight on the computation.  We will also need it for the\n",
+    "Futures API part of this example. Moreover, as this kind of computation\n",
+    "is often launched on super computer or in the Cloud, you will probably end\n",
+    "up having to start a cluster and connect a client to scale.  See \n",
+    "[dask-jobqueue](https://github.com/dask/dask-jobqueue),\n",
```

Comment:
Perhaps note that the specific library you want to use depends on your specific compute infrastructure?